src/opers.c

```diff
@@ -1796,6 +1796,9 @@ static inline Obj CacheOper(Obj oper, UInt i)
 
     if (cache == 0) {
         len = (i < 7 ? CACHE_SIZE * (i + 2) : CACHE_SIZE * (1 + 2));
+#ifdef HPCGAP
```

Comment:
Is it worth the complexity to save one word in non-HPC GAP. Why not just leave this word unused in regular GAP and save on code complexity. There might be a case for reorganising the method cache to  make more efficient use of the CPU cache, but that's a different problem